pkg/neorpc/result/version.go

```diff
@@ -104,5 +129,24 @@ func (p *Protocol) UnmarshalJSON(data []byte) error {
 	p.ValidatorsHistory = aux.ValidatorsHistory
 	p.InitialGasDistribution = fixedn.Fixed8(aux.InitialGasDistribution)
 
+	// Filter out unknown hardforks.
+	for _, hf := range aux.Hardforks {
+		if !config.IsHardforkValid(hf.Name) {
+			return fmt.Errorf("unexpected hardfork: %s", hf.Name)
+		}
+	}
+	// Keep them sorted.
+	p.Hardforks = make([]Hardfork, 0, len(aux.Hardforks))
+	for _, cfgHf := range config.Hardforks {
+		for _, auxHf := range aux.Hardforks {
+			if auxHf.Name == cfgHf.String() {
```

Comment:
We can support `HF_` for input here.

Reply:
Fixed.

Reply:
BTW, it's done the same way in https://github.com/neo-project/neo-modules/pull/838, it can handle both cases now.